mkdocs.yml

```diff
@@ -18,6 +18,7 @@ nav:
         - 'ILLIXR Glossary (Version 2)': glossary.md
         - 'ILLIXR API (Version 2)': api/html/annotated.html
         - 'ILLIXR License (Version 2)': LICENSE.md
+        - 'Contributing Guidelines (Version 2)': CONTRIBUTING.md
```

Comment:
Unrelated to this PR, but making a note: I don't like the `(Version 2)` after every heading. It doesn't look clean at all, and I'm not sure if it's required given that v2 is the default (plus, most people won't even know the difference between v1 and v2). My recommendation would be to remove all the version 2s. For version 1, we can say `Legacy Documentation (Version 1)`, and then remove `(Version 1)` from the individual lines.

Reply:
Agreed. New issue: #282.